random.cabal

```diff
@@ -63,14 +63,26 @@ build-type:         Custom
 extra-source-files:
     README.md
     CHANGELOG.md
+tested-with:         GHC == 7.10.2
+                   , GHC == 7.10.3
+                   , GHC == 8.0.2
+                   , GHC == 8.2.2
+                   , GHC == 8.4.3
+                   , GHC == 8.4.4
+                   , GHC == 8.6.3
+                   , GHC == 8.6.4
```

Comment:
```suggestion
```

8.6.3 and 8.6.4 are not enabled in `.travis.yml`.

Reply:
same here. It is not tested now, but it will be soon.